cellmlmanip/model.py

```diff
@@ -484,11 +549,7 @@ def graph(self):
                     graph.add_node(rhs, equation=None, variable_type=rhs.type)
                     graph.add_edge(rhs, lhs)
                 else:
-                    # this variable is a parameter - add to graph and connect to lhs
-                    rhs.type = 'parameter'
-                    dummy = self.add_number(rhs.initial_value, rhs.units)
-                    graph.add_node(rhs, equation=sympy.Eq(rhs, dummy), variable_type='parameter')
-                    graph.add_edge(rhs, lhs)
+                    assert False, 'Unexpected symbol {} on RHS'.format(rhs)  # pragma: no cover
```

Comment:
Not 100% sure but think the pragma should go on the line above?

Reply:
Nope, from the coverage report looks like this worked.